packages/frontend/src/lib/disk-image/DiskImageDetailsVirtualMachine.svelte

```diff
@@ -35,6 +35,16 @@ let notifySubscriber: Subscriber;
 const VM_LAUNCH_ERROR_MESSAGE = 'VM launch error';
 const GUIDE_LINK = 'https://github.com/containers/podman-desktop-extension-bootc/blob/main/docs/vm_launcher.md';
 
+// This is an issue with xterm not resizing properly when first initializing / loading
+// the terminal due to how we add padding. We must therefore call fit() approx 10ms after
+// initial loading to make sure the terminal is properly sized.
+// this is only called once when switching from no logs to logs
+$: if (noLogs === false) {
+  setTimeout(() => {
+    termFit?.fit();
+  }, 10);
+}
+
```

Comment:
same remark here

it makes me feel that if you copy/paste the same exact content inside 2 files, probably you can just use one component rather than duplicating the changes (so any fixes will help the other side rather than duplicating the changes (and at some point missing the enhancement/fix in one part)